app/Models/AlertSchedule.php

```diff
@@ -215,4 +213,12 @@ public function locations()
     {
         return $this->morphedByMany(\App\Models\Location::class, 'alert_schedulable', 'alert_schedulables', 'schedule_id', 'alert_schedulable_id');
     }
+
+    public function __toString()
+    {
+        return ($this->recurring ?
+            'Recurring Alert Schedule (' . implode(',', $this->recurring_day) . ') ' :
```

Comment:
`$this->recurring_day` is a string already?

Reply:
Not when using the Accessor to access it.